src/server/biosecurity-map/upload-config.js

```diff
@@ -0,0 +1,4 @@
+export const uploadConfig = {
+  sectionKey: 'biosecurity-map',
+  questionKey: 'upload-plan'
```

Comment:
I can see why you want to have the section key defined somewhere to be reused across all pages in the section but the question key is only used in one page so I don't think we need it here?

Reply:
You seem to be using it in two pages, the same question key, is that needed because of the upload progress page?

Reply:
for file upload its actually used in 2 pages

Reply:
the actual upload, and the progress page